src/CoreText/CTFontManager.cs

```diff
@@ -45,28 +45,32 @@ namespace CoreText {
 	public enum CTFontManagerScope : uint {
 		None = 0,
 		Process = 1,
-#if !NET
-		[iOS (13,0)][TV (13,0)][Watch (6,0)]
-#else
+#if NET
 		[SupportedOSPlatform ("ios13.0")]
 		[SupportedOSPlatform ("tvos13.0")]
+#else
+		[iOS (13,0)]
+		[TV (13,0)]
+		[Watch (6,0)]
 #endif
 		Persistent = 2,
-#if !NET
-		[NoiOS][NoTV][NoWatch]
-#else
-		[UnsupportedOSPlatform ("tvos")]
+#if NET
 		[UnsupportedOSPlatform ("ios")]
-		[UnsupportedOSPlatform ("maccatalyst")]
```

Comment:
Do we want to remove these maccatalyst's?

Reply:
I assume that this framework is just not supported in maccatalyst?

Reply:
There is no [NoMacCatalyst] on the legacy attributes, and it should be implied by `[UnsupportedOSPlatform ("ios")]`

Reply:
Turns out that xtro did not like this removal